css/style.css

```diff
@@ -0,0 +1,485 @@
+@import url('./fonts.css');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.level, .play {
+  cursor: pointer;
+}
+
+.play {
+  margin-top: 88px;
+  min-width: 352px;
+  min-height: 95px;
+  background: #D62B2E;
```

Comment:
Codacy found an issue: [Expected "#D62B2E" to be "#d62b2e" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)